test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/ArrayListTest.scala

```diff
@@ -37,4 +469,13 @@ class ArrayListFactory extends AbstractListFactory {
 
   override def empty[E: ClassTag]: ju.ArrayList[E] =
     new ju.ArrayList[E]
+
+  def from[E](coll: ju.Collection[E]): ju.ArrayList[E] =
+    new ju.ArrayList[E](coll)
```

Comment:
This should actually override `def fromElements` from `ListFactory` in `ListTest.scala`. Then we get plenty of tests for free.